.github/actions/generic-ci/action.yml

```diff
@@ -103,14 +114,14 @@ runs:
         -Werror=dev
         -Werror=deprecated
         --warn-uninitialized
-        -DBUILD_TESTING=ON
+        -DBUILD_TESTING=${{ inputs.cpu == 'x86_64' && 'ON' || 'OFF' }}
         -DCMAKE_BUILD_TYPE=Release
         -DCMAKE_INSTALL_PREFIX:PATH=../install
         -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON
         -DCMAKE_POLICY_DEFAULT_CMP0091=NEW
         -DCMAKE_PREFIX_PATH:PATH=$(pwd)/../dependencies/install/
-        -DF3D_BINDINGS_JAVA=ON
-        -DF3D_BINDINGS_PYTHON=ON
+        -DF3D_BINDINGS_JAVA=${{ inputs.cpu == 'x86_64' && 'ON' || 'OFF' }}
+        -DF3D_BINDINGS_PYTHON=${{ inputs.cpu == 'x86_64' && 'ON' || 'OFF' }}
```

Comment:
any reason why the bindings cannot be supported on arm ?

Reply:
right, the dependencies. Do you know if python bindings will work in the superbuild ?

Reply:
Maybe it works. `pybind11` is a header-only library. I'll try to enable it.

Reply:
ok, too bad we cant test it in the CI, but well, not going to build python ourselve here

Reply:
Well, it works :)  
Not tested tho